production/helm/templates/loki/podsecuritypolicy.yaml

```diff
@@ -23,10 +23,7 @@ spec:
   runAsUser:
     rule: 'MustRunAsNonRoot'
   seLinux:
-    rule: 'MustRunAs'
```

Comment:
Sorry, my cluster doesn't use seLinux so couldn't test this. Seems `MustRunAs` is a valid option but it requires setting a `securityContext.seLinuxOptions`. Not sure the impact of that so it's fine to revert to `RunAsAny`.

Reply:
Changes made.